File: AlumniDarli/app/Models/ContactMessage.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class ContactMessage extends Model
{
    use HasFactory;
    protected $fillable = [
        'user_id',
        'subject',
        'message',
        'attachment',
        'admin_reply',
        'replied_at',
        'status',
    ];

    protected $casts = [
        'replied_at' => 'datetime',
    ];

    public function user()
    {
        return $this->belongsTo(User::class, 'user_id', 'id_user');
    }
}

File: Darli/app/src/main/java/com/example/darli/data/model/ContactMessage.kt
package com.example.darli.data.model

data class ContactMessage(
    val id: Int,
    val subject: String,
    val message: String,
    val attachment: String?,
    val admin_reply: String?,
    val replied_at: String?,
    val created_at: String?
)

data class ContactMessageResponse(
    val response_code: Int,
    val content: List<ContactMessage>
)
